README.md

```diff
@@ -287,12 +287,16 @@ We need to have two separate Meteor Up projects. For that, create two directorie
 
 In the staging `mup.js`, add a field called `appName` with the value `staging`. You can add any name you prefer instead of `staging`. Since we are running our staging app on port 8000, add an environment variable called `PORT` with the value 8000.
 
-You might also have to tell docker to use this custom port like this:
+You also have to tell meteor to use this custom port like this:
```

Comment:
Not if you change the meteor ENV settings, which is what the updated documentation does. If you just put port out of env, it does nothing.